packages/database/schema.ts

```diff
@@ -67,6 +67,8 @@ export const accounts = mysqlTable(
     createdAt: timestamp('created_at').$defaultFn(() => new Date()),
     lastLoginAt: timestamp('last_login_at'),
     passwordHash: varchar('password_hash', { length: 255 }),
+    recoveryEmailHash: varchar('hashed_recovery_email', { length: 255 }),
+    recoveryEmailRecoveredAt: timestamp('recovery_email_recovered_at'),
```

Comment:
this field is named confusingly, name it something like `recovery_mail_verified_at`, then you can delete the migration files and entry from journal and regenerate them